.docker/php-fpm/7.1/Dockerfile

```diff
@@ -10,6 +10,7 @@ RUN apt-get update -qq \
     && echo "zend_extension=$(find /usr/local/lib/php/extensions/ -name xdebug.so)" > /usr/local/etc/php/conf.d/xdebug.ini \
     && echo "xdebug.remote_enable=on" >> /usr/local/etc/php/conf.d/xdebug.ini \
     && echo "xdebug.remote_autostart=off" >> /usr/local/etc/php/conf.d/xdebug.ini \
+    && echo "xdebug.remote_host=XDEBUG_REMOTE_HOST" >> /usr/local/etc/php/conf.d/xdebug.ini \
```

Comment:
As it's a specific configuration it's better to add a specific file like

ADD config/custom-php.ini /usr/local/etc/php/conf.d/custom-php.ini

Reply:
It's in case you need the DNS for the host for something else (no more need for the external_host with the ip of the host, you can use the docker internal network)